hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1235,6 +1241,7 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 		podMonitor.Spec.Selector = *controlPlaneOperatorDeployment.Spec.Selector
 		podMonitor.Spec.PodMetricsEndpoints = []prometheusoperatorv1.PodMetricsEndpoint{{Port: "metrics"}}
 		podMonitor.Spec.NamespaceSelector = prometheusoperatorv1.NamespaceSelector{MatchNames: []string{controlPlaneNamespace.Name}}
+		podMonitor.Spec.JobLabel = "app"
```

Comment:
why? This will be very confusing if we have `app=namespace/podmonitor/$name` rather than `job=namespace/podmonitor/$name` or not?